### PR TITLE
Change name of the UVM PR build to include UVM Off

### PR DESCRIPTION
**Pull request by ZUUL42**
@trilinos/framework

## Motivation

Naming the UVM = OFF build as `Trilinos_pullrequest_cuda_10.1.105_uvm` is a bit of a misnomer as it sounds like the opposite, UVM = On. #9068 So, to avoid confusion, this will change the name to `Trilinos_pullrequest_cuda_10.1.105_uvm_off` in the code.
Once the PR is tested and reviewed, I'll merge it manually as to time it with changing of the job name itself.

### CODE REVIEW

**Review by prwolfe — APPROVED** [review]

**Comment by ZUUL42** [comment]
Jenkins job name changed and name in master merge PR autotester config changed.
I'll wait until it passes a nightly Master Merge PR before adding it to the PR build autotester config.

**Comment by ZUUL42** [comment]
Master Merge PR successful.
UVM Off build added to PR testing.

**Pull request merged**
